index.html

```diff
@@ -5,16 +5,23 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
     <title>Awesome books</title>
 </head>
 <body>
-    <h1>Awesome books</h1>
-    <ul class="books-list"></ul>
-    <form action="#" class="form">
-        <input type="text" id="title" placeholder="Title" required><br><br>
-        <input type="text" id="author" placeholder="Author" required><br><br>
-        <button  type="submit" class="add-book-btn">Add</button>
-    </form>
+  <h1>All Awesome books</h1>
+    <section class="home" id="show-book">
+      <ul class="books-list"></ul>
+    </section>
+    <section class="form-container">
+      <h2>Add a new book</h2>        
+      <form action="#" class="form">
+          <input type="text" id="title" placeholder="Title" required><br><br>
+          <input type="text" id="author" placeholder="Author" required><br><br>
+          <!-- <p id="error"> </p> -->
```

Comment:
- [ ]  Please kindly remove this old piece of commented code so the repo gets cleaner 💪